doc/notes/CZT.rst

```diff
@@ -35,7 +35,7 @@ Implementation Notes
 :py:func:`~pyffs.czt.czt` can be used to compute :math:`\text{CZT}_{N}^{M}` as
 defined above, with :math:`L \ge N + M - 1` optimally chosen.
 
-:py:func:`~pyffs.czt.cztn` can be used to compute :math:`\text{CZT}_{N_1 \times N_2 \times
-\cdots N_D}^{M_1, M_2, \ldots, M_D}`, with :math:`L_d \ge N_d + M_d - 1` optimally chosen for
+:py:func:`~pyffs.czt.cztn` can be used to compute :math:`\text{CZT}_{N_1, N_2,
+\ldots, N_D}^{M_1, M_2, \ldots, M_D}`, with :math:`L_d \ge N_d + M_d - 1` optimally chosen for
 :math:`d = 1, 2, \ldots, D`. In our implementation, we opt for a more efficient approach than
 applying the 1D CZT along each dimension.
```

Comment:
I forgot to put it here, but perhaps we can put CZT in math mode? For consistency as I saw you did that (and followed suit) in the interpolation page.

Reply:
You mean for line 41?

Reply:
I agree.